client/my-sites/customer-home/cards/tasks/style.scss

```diff
@@ -0,0 +1,30 @@
+.tasks.action-panel {
+	.tasks__timing {
+		display: flex;
+
+		.gridicon {
+			margin-right: 4px;
+		}
+	}
+
+	.action-panel__title {
+		font-family: 'Recoleta', serif;
```

Comment:
Now that the Recoleta font is in My Home (#41045), it should show up if you rebase this branch.